requirements.txt

```
@@ -12,6 +12,7 @@ httpx>=0.25.0
 instructlab-quantize>=0.1.0
 instructlab-schema>=0.2.0
 jsonschema>=4.21.1
+instructlab-sdg>=0.0.2,<0.1.0
```

```suggestion
instructlab-sdg>=0.0.2
jsonschema>=4.21.1
```

Moved up a line to sort properly and also removed the upper range in line with recent changes to most dependencies.